test.sh.in

```diff
@@ -218,20 +230,31 @@ setupVirtualEnv(){
 
 BASE_REPO_URL="${BASE_REPO_URL}"
 runTests() {
-  read array<<<"${TEST_GROUP}"
-  args=()
-  group=" --group="
-  for item in $array; do
-    args+="${group}${item}"
-  done
-  cp -f ${WORKSPACE}/build-config/config.ini ${WORKSPACE}/RackHD/test/config
-  cd ${WORKSPACE}/RackHD/test
-  RACKHD_BASE_REPO_URL=${BASE_REPO_URL} RACKHD_TEST_LOGLVL=INFO \
-      python run.py ${args} --with-xunit 
-  mkdir -p ${WORKSPACE}/xunit-reports
-  cp *.xml ${WORKSPACE}/xunit-reports
+  if [ "$RUN_FIT_TEST" == true ] ; then
+     cd ${WORKSPACE}/RackHD/test
+     #TODO Parameterize FIT args
+     python run_tests.py -test deploy/rackhd_stack_init.py -stack vagrant  -port 9090 -xunit
+     python run_tests.py -test tests/ -group $TEST_GROUP -stack vagrant -port 9090 -v 9 -xunit
+     mkdir -p ${WORKSPACE}/xunit-reports
```

Comment:
maybe can move these two common lines out of if-else block.
moreover, if getting into runTests() function, and both FIT/CIT flag are false, there should be an error-check.
otherwise,  jenkins job will be green but actually bad.